example/spring-batch/src/main/kotlin/com/linecorp/kotlinjdsl/example/spring/batch/entity/book/Isbn.kt

```diff
@@ -0,0 +1,15 @@
+package com.linecorp.kotlinjdsl.example.spring.batch.entity.book
+
+import jakarta.persistence.Column
+import jakarta.persistence.Embeddable
+import java.io.Serializable
+
+/**
+ * International Standard Book Number
+ */
+@Embeddable
+data class Isbn(
+    @Column(name = "isbn")
+    val value: String,
+) : Serializable
+
```

Comment:
You can use ktlintFormat for applying format

Reply:
I fixed code format applying ktlintFormat.